src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const tr = document.querySelectorAll('tr');
```

Comment:
Don't cut variable names
```suggestion
const tableRow = document.querySelectorAll('tr');
```